packages/SwingSet/docs/vat-upgrade.md

```diff
@@ -94,32 +100,33 @@ To enable upgrade, the v1 code must stash important data in durable storage, and
 
 For upgrade-important in-RAM objects, the v1 code needs to store a "reattachment handle" (name and API TBD) in durable storage. This gives the v2 code the authority to take over the identity of these exported objects.
 
-For upgrade-important virtual objects (specifically durable virtual objects), the v1 code must store the "durable kind handle" in durable storage. This handle is created before calling `makeDurableKind`, like this:
+For upgrade-important durable objects, the v1 code must store the "durable kind handle" in durable storage. This is done automatically by `prepareExoClass()` and `prepareExoClassKit()`, but can also be done manually (see [Virtual and Durable Objects](https://github.com/Agoric/agoric-sdk/blob/master/packages/SwingSet/docs/virtual-objects.md#virtual-and-durable-objects)).
```

Comment:
why an absolute URL here? i see relative URLs for most links within swingset docs.

Reply:
Good point; updated.